### ISSUE TITLE
Add basic if/else logic in template placeholders

### ISSUE THREAD

**Issue by dafky2000** [dafky2000/simplectemplate#23]
Right now we simply search for {{data.varname}} and replace with defined. Would be nice to have something like {{data.varname:replace with this text if varname is defined}}.

Than we could do something like:

```html
{{data.link1:<a href="staticlink.html">Test link</a>}}
```

Kind of useless right now but is required for some advanced functionality defined in #22

**Comment by dafky2000** [comment]
Instead of use the colon `:` as the separation, it is safe to assume that the first space separates the content from the condition.

{{data.key text to use if data.key is defined and non-empty}}

**Comment by dafky2000** [comment]
Since we are going to try to support Mustache syntax than the above should be valid as well as:
```
{{#data.key}}
   Some data if data.key has a value
{{/data.key}}
```

This will likely be under some option where we can define the opening and closing "braces" to include the data variable name.

**Comment by dafky2000** [comment]
Closed via #61
